capstone-frontend/src/components/MeetingCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { Button, Card, Col } from 'react-bootstrap';
+import moment from 'moment';
+
+import '../styles/Groups.css';
+
+
+class MeetingCard extends Component {
```

Comment:
Add `export` here and make this a named import instead of default

Reply:
Will do!